_redirects

```diff
@@ -263,4 +263,5 @@
 /reference/runs	/dbt-cloud/api	302
 /reference/using-the-dbt-cloud-api	/dbt-cloud/api	302
 https://tutorial.getdbt.com/*	https://docs.getdbt.com/:splat	301!
-reference/model-selection-syntax/#test-selection-examples    /reference/node-selection/test-selection-examples 302
```

Comment:
Does this imply that there are bugged links out there?.

Reply:
I'm not sure — I just noticed that this was missing a `/` so probably wasn't working in the first place. I was opportunistic in fixing it :)